src/modules/navigator/mission_block.cpp

```diff
@@ -71,6 +74,13 @@ MissionBlock::~MissionBlock()
 bool
 MissionBlock::is_mission_item_reached()
 {
+	if (_mission_item.nav_cmd == NAV_CMD_DO_SET_SERVO) {
+
+		up_pwm_servo_arm(true);
```

Comment:
This is crossing the abstraction layers and not something we should be generally doing. The right approach would be to publish to the actuator_control group #2 which is for payload, and to add a mixer in your airframe config which maps this to the output. The reason is that we want our architecture to scale to PWM but also UAVCAN.
https://pixhawk.org/dev/mixing#control_groups